vterm/ops.go

```diff
@@ -160,6 +160,18 @@ func (v *VTerm) shiftCursorY(diff int) {
 	v.setCursorPos(v.Cursor.X, v.Cursor.Y+diff)
 }
 
+func (v *VTerm) setChar(x, y int, r rune, style ecma48.Style) {
```

Comment:
`style ecma48.Style` could probably be removed since it look like `v.Cursor.Style` is used for all the setChars (`style` could be set to `v.Cursor.Style`).